Formula/libmatio.rb

```diff
@@ -42,6 +42,18 @@ def install
         if (!(matfp = Mat_Open(argv[1], MAT_ACC_RDONLY)))
           abort();
         Mat_Close(matfp);
+
+        mat = Mat_CreateVer("test_writenan.mat",NULL,mat_file_ver);
```

Comment:
This test cannot pass, whether on macOS or Linux or anywhere else. `mat`, `mat_file_ver`, `matvar` (below), `dims` and `data` are not defined anywhere.